src/main/scala/akka/stream/contrib/UnfoldFlow.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.contrib
+
+import akka.stream.{Attributes, FanOutShape2}
+import akka.stream.stage.{GraphStage, GraphStageLogic, OutHandler}
+import akka.util.Timeout
+
+/** INTERNAL API */
+private[akka] abstract class UnfoldFlowGraphStageLogic[O, S, E] private[stream] (shape: FanOutShape2[O, S, E],
+                                                                                 seed: S,
+                                                                                 timeout: Timeout)
```

Comment:
Wow, such indent... can we move things to a new line or does scalafmt not allow that?

Reply:
I used the same formatting rules as in Alpakka and do not want to diverge from those.